### PR TITLE
add tmt_quant_parser parameters

### PR DESCRIPTION
**Pull request by MKoesters**

### CODE REVIEW

**Review by ArtiVlasov — APPROVED** [review]

File: jsons/parameters.json [hunk 1]
```diff
@@ -7926,6 +7926,7 @@
             "pipi_style_1": "header_translations",
             "ptmshepherd_style_1": "header_translations",
             "tag_graph_style_1": "header_translations",
+            "tmt_quant_style_1": "header_translations",
```
Comment (fu) [inline jsons/parameters.json:7929]: should probably be cz_tmt_quant_style_1

File: jsons/styles.json [hunk 2]
```diff
@@ -783,6 +783,14 @@
             "1.0.0"
         ]
     },
+    {
+        "name": "tmt_quant_style_1",
+        "reference": "Ursgal2 Team",
```
Comment (fu) [inline jsons/styles.json:788]: Should be reference to Cellzome team. I think that would be Mathieson, T., Sweetman G. and Fufezan. C. Better check with Toby

File: jsons/styles.json [hunk 3]
```diff
@@ -783,6 +783,14 @@
...
+        "style": "tmt_quant_style_1",
```
Comment (fu) [inline jsons/styles.json:789]: and style ofcoz as above

**Review by fu — CHANGES_REQUESTED** [review]
see comments

**Pull request merged**